src/compiler/program.ts

```diff
@@ -241,22 +241,27 @@ namespace ts {
         return errorMessage;
     }
 
-    const redForegroundEscapeSequence = "\u001b[91m";
-    const yellowForegroundEscapeSequence = "\u001b[93m";
-    const blueForegroundEscapeSequence = "\u001b[93m";

```

Comment:
Note: the blue sequence was the same as yellow before. I'm guessing that's a behavior bug that was never exposed. I've removed the sequence but left the behavior of using the yellow color in `getCategoryFormat`.

Reply:
> Note: the blue sequence was the same as yellow before. I'm guessing that's a behavior bug that was never exposed.

We don't actually use `ErrorCategory.Warning` right now, so it never got noticed/changed until it mattered, like for https://github.com/Microsoft/TypeScript/pull/19126